app/views/layouts/application.html.erb

```diff
@@ -169,183 +169,23 @@
   <% end %>
 
   <a class="skip-to-link" href="#main"><%= t('skip_links.to_main') %></a>
-  <a class="skip-to-link" href="<%= t('skip_links.to_accessibility_link') %>"><%= t('skip_links.to_accessibility') %></a>
+  <a class="skip-to-link" href="<%= t('skip_links.to_accessibility_link') %>">
+    <%= t('skip_links.to_accessibility') %>
+  </a>
 
   <%= render 'shared/svg_icons' %>
   <%= render 'shared/header' %>
-
-  <div class="l-context-bar">
-    <div class="l-constrained">
-      <nav class="breadcrumbs" role="navigation" aria-label="breadcrumbs">
-        <ul class="unstyled-list">
-          <li class="breadcrumbs__item" itemscope="" itemtype="http://data-vocabulary.org/Breadcrumb">
-            <a class="breadcrumbs__link" href="<%= t('mas_home_url') %>" itemprop="url">
-              <span itemprop="title"><%= t('breadcrumbs.home') %></span>
-            </a>
-          </li>
-          <li class="breadcrumbs__item" itemscope="" itemtype="http://data-vocabulary.org/Breadcrumb">
-            <span class="visually-hidden">&gt;</span>
-            <span class="breadcrumbs__divider" aria-hidden="true"></span>
-            <a class="breadcrumbs__link" href="<%= t('breadcrumbs.pension_and_retirement_url') %>" itemprop="url">
-              <span itemprop="title"><%= t('breadcrumbs.pension_and_retirement') %></span>
-            </a>
-          </li>
-          <li class="breadcrumbs__item" itemscope="" itemtype="http://data-vocabulary.org/Breadcrumb">
-            <span class="visually-hidden">&gt;</span>
-            <span class="breadcrumbs__divider" aria-hidden="true"></span>
-            <a class="breadcrumbs__link" href="<%= t('breadcrumbs.financial_advice_url') %>" itemprop="url">
-              <span itemprop="title"><%= t('breadcrumbs.financial_advice') %></span>
-            </a>
-          </li>
-          <li class="breadcrumbs__item" itemscope="" itemtype="http://data-vocabulary.org/Breadcrumb">
-            <span class="visually-hidden">&gt;</span>
-            <span class="breadcrumbs__divider" aria-hidden="true"></span>
-            <a class="breadcrumbs__link" href="<%= t('breadcrumbs.directory_url') %>" itemprop="url">
-              <span itemprop="title"><%= t('breadcrumbs.directory') %></span>
-            </a>
-          </li>
-        </ul>
-      </nav>
-    </div>
-  </div>
+  <%= render 'shared/breadcrumbs' %>
```

Comment:
Since you're refactoring ... IMHO, it is better never to use 'shared' folder because if exists on app everything will be moved to there. I would use the views structure more in a resource way (e.g breadcrumbs/show). Wdyt?